packages/react-components/react-theme/src/themes/ThemeMotion.stories.tsx

```diff
@@ -0,0 +1,118 @@
+import * as React from 'react';
+import { teamsLightTheme, DurationTokens, CurveTokens } from '../index';
+import { makeStyles, shorthands } from '@fluentui/react-components';
+
+export default {
+  title: 'Theme',
+};
+
+const theme = teamsLightTheme;
+
+const useStyles = makeStyles({
+  durationAnimation: {
+    width: '4px',
+    height: '64px',
+    ...shorthands.margin(0, '30px'),
+    backgroundColor: '#ccc',
+    position: 'relative',
+    animationIterationCount: 'infinite',
+    animationName: {
+      from: { transform: 'rotate(0)' },
+      to: { transform: 'rotate(180deg)' },
+    },
+  },
+  curvesAnimation: {
+    width: '64px',
+    height: '64px',
+    // ...shorthands.margin(0, '30px'),
+    ...shorthands.borderRadius('64px'),
+    backgroundColor: '#ccc',
+    position: 'relative',
+    animationIterationCount: 'infinite',
+    animationName: {
+      from: { left: 0 },
+      to: { left: '200px' },
+    },
+  },
+});
+
+export const MotionDuration = () => {
```

Comment:
for both stories to be exportable in codesandbox, they need to be in their own separate modules

Reply:
That applies to other theme stories as well.
I would not expect anybody to export these to codesandbox. Should that requirement arise, let's refactor all the theme stories together.